src/plugins/inputs/topic.js

```diff
@@ -14,9 +15,6 @@ exports.input = function(network, chan, cmd, args) {
 
 		return;
 	}
-
-	var irc = network.irc;
-	irc.raw("TOPIC", chan.name, args.join(" "));
-
+	client.setTopic(chan.name, args.join(" "));
```

Comment:
Need to change on L10/R11 as well then. I'll approve after that and let @xPaw decide if he cares or not.